### PR TITLE
feat(j-s): COA ruling in form stepper

### PR DESCRIPTION
**Pull request by oddsson**
# COA ruling in form stepper

[Asana](https://app.asana.com/0/1199153462262248/1205494465376374/f)

## What

Show COA ruling in the form stepper. Before this change the form stepper would show "Nidurstada", even if COA had made a ruling.

## Why

We are showing the ruling in the form stepper for other rulings.

## Screenshots / Gifs

Screenshot 2023-10-03 at 13 58 20

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/utils/hooks/useStringHelpers/useStringHelpers.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { sections } from '@island.is/judicial-system-web/messages'
+
+import { useStringHelpersStrings as strings } from './useStringHelpers.strings'
+
+/**
+ * This hook contains helper functions for strings. The reason for it
+ * being a hook, instead of a simple utility function, is that it
+ * it uses hooks internally, f.x. the useIntl hook.
+ */
+const useStringHelpers = () => {
```
Comment (unakb) [inline apps/judicial-system/web/src/utils/hooks/useStringHelpers/useStringHelpers.ts:13]: Nice!

File: apps/judicial-system/web/src/components/TagAppealState/TagAppealState.tsx [hunk 2]
```diff
@@ -64,43 +66,18 @@ const TagAppealState: React.FC<React.PropsWithChildren<Props>> = ({
...
-        }
+      return {
+        color:
+          ruling === CaseAppealRulingDecision.ACCEPTING
+            ? 'mint'
+            : ruling === CaseAppealRulingDecision.CHANGED ||
+              ruling === CaseAppealRulingDecision.REPEAL
+            ? 'rose'
+            : 'blueberry',
+        text: getAppealResultText(ruling),
```
Comment (unakb) [inline apps/judicial-system/web/src/components/TagAppealState/TagAppealState.tsx:77]: Nice cleanup, much more readable 🤩

**Review by unakb — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -110,6 +110,7 @@
     "@sindresorhus/slugify": "1.0.0",
     "@statoscope/webpack-plugin": "5.20.1",
     "@storybook/core-server": "7.4.1",
+    "@testing-library/react-hooks": "8.0.1",
```
Comment (saevarma) [inline package.json:113]: This library shouldn't be required anymore
https://github.com/testing-library/react-hooks-testing-library#a-note-about-react-18-support

And your spec file is actually importing
`import { renderHook } from '@testing-library/react'`

**Review by saevarma — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (unakb) [reply package.json]: Removed ✅

**Pull request merged**